subsys/bluetooth/host/sdp.c

```diff
@@ -2464,7 +2479,36 @@ int bt_sdp_get_proto_param(const struct net_buf *buf, enum bt_sdp_proto proto,
 		return res;
 	}
 
-	res = sdp_get_uuid_data(&attr, &pd, proto);
+	res = sdp_get_uuid_data(&attr, &pd, proto, 1U);
+	if (res < 0) {
+		BT_WARN("Protocol specifier 0x%04x not found, err %d", proto,
+			res);
+		return res;
+	}
+
+	return sdp_get_param_item(&pd, param);
+}
+
+int bt_sdp_get_addl_proto_param(const struct net_buf *buf, enum bt_sdp_proto proto,
+			   uint8_t param_index, uint16_t *param)
```

Comment:
Looks like you have some issues with getting the indentation right. Have you set your editor to have tab=8 spaces (as we use that in Zephyr)? 

```suggestion
int bt_sdp_get_addl_proto_param(const struct net_buf *buf, enum bt_sdp_proto proto,
				uint8_t param_index, uint16_t *param)
```

Reply:
Updated，I remove one tab in case the line exceed the max length.